AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -15,6 +15,7 @@
 !
 ! SECTION: Popups - Regular rules
 !
+store.steampowered.com#%#//scriptlet('trusted-click-element', '.broadcast_embed_top_ctn_trgt button:has(> svg.SVGIcon_X)', '', '500')
```

Comment:
Cannot reproduce broadcast. @AdguardTeam/filters-maintainers could you please take a look?

Reply:
Thank you, reproduced.